streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -903,6 +913,15 @@ void shutdown(final boolean clean) {
                         tasksToCloseClean.remove(task);
                     }
                 }
+            } catch (final TaskTimeoutExceptions taskTimeoutExceptions) {
```

Comment:
During shutdown, we don't need to trigger `task.timeout.ms` but can re-throw the `TimeoutException` to trigger a "dirty close" instead.